.github/workflows/CI.yaml

```diff
@@ -20,3 +20,13 @@ jobs:
       - name: Run Tests
         run: |
           . (Join-Path "." "Tests/runtests.ps1")
+  
+  TestALGOActionOrigins:

```

Comment:
Why not just create another step in the Test job - instead of spinning up another runner, checking out all source code and running this in parallel

Reply:
or actually - since you placed the test in the tests folder and celled it xxx.test.ps1 - it should already be executed by the runtests command above - is this job needed at all`?

Reply:
That's true. The idea was to split this from the others since it's not actually testing a specific action. Thus, in some PRs (like https://github.com/microsoft/AL-Go/pull/382) I'm okay with this test failing right up until I want to merge
I see your point though, so I'll remove the second step :) We can always change it later if we feel like it